pipe-cli/pipe.py

```diff
@@ -855,13 +855,19 @@ def storage_move_item(source, destination, recursive, force, exclude, include, q
 @click.option('-l', '--file-list', required=False, help="Path to file with file paths that should be copied. This file "
                                                         "should be tub delimited and consist of two columns: "
                                                         "relative path to file and size.")
+@click.option('-sl', '--symlinks', required=False, default="follow",
```

Comment:
Can't we use [Choice](https://click.palletsprojects.com/en/7.x/options/#choice-options) to simplify allowed values handling?

Reply:
thanks, added